js/src/Zone.js

```diff
@@ -0,0 +1,57 @@
+import ExpectedMark from "./ExpectedMark";
+import UXCapture from "./UXCapture";
+
+// array holding current, global set of expected zones
+let expectedZones = [];
+
+/**
+ * Zone represents collection of elements groupped together and corresponding phase of page load
+ *
+ * Only one set of zones can be tracked for the same view at one point in time
+ */
+export default class Zone {
+  static setExpectedZones(zones) {
```

Comment:
This method feels like something that would be the responsibility of a Zone 'controller', not the `Zone` class itself - I think a `View` abstraction would be a good fit - each `View` instance would have its own set of `expectedZones`, or more simply, each `View` would have `this.zones` - you wouldn't need to keep re-assigning the `expectedZones` variable.